tests/utils/TestHelper.ts

```diff
@@ -0,0 +1,242 @@
+import Str from 'expensify-common/lib/str';
+import Onyx from 'react-native-onyx';
+import CONST from '@src/CONST';
+import * as Session from '@src/libs/actions/Session';
+import HttpUtils from '@src/libs/HttpUtils';
+import * as NumberUtils from '@src/libs/NumberUtils';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {PersonalDetails, Report} from '@src/types/onyx';
+import waitForBatchedUpdates from './waitForBatchedUpdates';
+
+type MockFetch = ReturnType<typeof jest.fn> & {pause?: () => void; fail?: () => void; succeed?: () => void; resume?: () => Promise<void>};
```

Comment:
Also maybe we can use just `jest.Mock & ` instead of ` ReturnType<typeof jest.fn> &`
```suggestion
type MockFetch = ReturnType<typeof jest.fn> & {
  pause?: () => void;
  fail?: () => void;
  succeed?: () => void; 
  resume?: () => Promise<void>
};
```